documentation/src/main/asciidoc/introduction/Introduction.adoc

```diff
@@ -542,7 +542,7 @@ public List<Book> findBooks(String titlePattern) {
 }
 ----
 
-You might be thinking that our query method looks a bit boilerplatey.
+You might be thinking that our query method looks a bit boilerplate.
```

Comment:
This was intentional. I'm allowed to invent words like "boilerplatey" because they're run and I like saying them :)

Reply:
ok, I googled a lot and no wonder I didn't find it anywhere, :).

Reply:
My original thought is if it is a new adjective word, it might be "boilerplaty" instead of "boilerplatey"? (inasmuch as 'taste' vs 'tasty'), but maybe you intend some fun here. Anyway, I revert back this change, among others.